docs/test/test.md

```diff
@@ -60,8 +67,8 @@ make user-agent-start
 
 ## Agents
 
-- To access user agent wasm open [user home page](https://localhost:8091/wallet/dashboard).
-- To access second user agent wasm open [user home page](https://localhost:8071/wallet/dashboard).
+- To access user agent wasm open [user home page](https://user.ui.agent.example.com:8091/dashboard).
+- To access second user agent wasm open [user home page](https://second.ui.user.agent.example.com:8071/dashboard).
```

Comment:
We should add a note here saying the user needs to open the second agent in a different browser (not just tab) so the login will work as expected after #497 is fixed.

If their browser supports containers, they can use the same browser but open the login page in different containers.

Reply:
I will add it.